python/console/console.py

```diff
@@ -314,6 +320,19 @@ def __init__(self, parent=None):
         self.runScriptEditorButton.setToolTip(runScriptEditorBt)
         self.runScriptEditorButton.setText(runScriptEditorBt)
 
+        # Action Run Selected
+        runSelectedEditorBt = QCoreApplication.translate("PythonConsole", "Run Selected")
+        self.runSelectedEditorButton = QAction(self)
+        self.runSelectedEditorButton.setCheckable(False)
+        self.runSelectedEditorButton.setEnabled(True)
+        self.runSelectedEditorButton.setIcon(
+            QgsApplication.getThemeIcon("mActionRunSelected.svg")
+        )
+        self.runSelectedEditorButton.setMenuRole(QAction.MenuRole.PreferencesRole)
+        self.runSelectedEditorButton.setIconVisibleInMenu(True)
+        self.runSelectedEditorButton.setToolTip(runSelectedEditorBt + " <b>Ctrl+e</b>")
```

Comment:
You could set the shortcut directly for this action here, and then remove the runScut QShortcut from console_editor.py. That'd be cleaner (and easier to maintain!)